src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -39,6 +39,9 @@
 
         az vm create -g group-name -n vm-name --admin-username deploy  \\
           --image debian --secrets "$vm_secrets"
+  - name: Create a Key Vault certificate. (autogenerated)
+    text: "az keyvault certificate create --name cert1 \\\n  --policy \"$(az keyvault certificate get-default-policy)\" --vault-name vaultname"
```

Comment:
What's up with the `\\\n`?

Reply:
Those are escape and newline characters for multiline examples

Reply:
They shouldn't be present in the text.